File: zxing-android-embedded/src/com/journeyapps/barcodescanner/camera/CameraParametersCallback.java
package com.journeyapps.barcodescanner.camera;

import android.hardware.Camera;

/**
 * Callback for {@link Camera.Parameters}.
 */
public interface CameraParametersCallback {

    /**
     * Changes the settings for Camera.
     *
     * @param parameters {@link Camera.Parameters}.
     * @return {@link Camera.Parameters} with arguments.
     */
    Camera.Parameters changeCameraParameters(Camera.Parameters parameters);
}

File: zxing-android-embedded/src/com/journeyapps/barcodescanner/camera/CameraParametersCallback.kt
package com.journeyapps.barcodescanner.camera

import android.hardware.Camera

/**
 * Callback for [Camera.Parameters].
 */
interface CameraParametersCallback {
    /**
     * Changes the settings for Camera.
     *
     * @param parameters [Camera.Parameters].
     * @return [Camera.Parameters] with arguments.
     */
    fun changeCameraParameters(parameters: Camera.Parameters): Camera.Parameters
}